commit-to-output.js

```diff
@@ -39,6 +39,11 @@ export const formatType = {
 
 function toStringPlaintext (data) {
   let s = ''
+  if (data.cveId) {
+    const pr = data.pr ? data.prUrl : ''
+    return ` * ${data.cveId} - ${data.summary.trim()} - ${pr}`
```

Comment:
I don't think this is what you want, it doesn't include the semver or any of the other things here, how about you just do this instead:

```js
s += `(${data.cveID}) `
```